tutorials/ase_database/generate_input.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+"""
+Title: generate_input.py
+
+Description
+-----------
+    Creates CATMAP input table from ase databases.
+
+Input
+----
+    file_name : string
+        Output filename, e.g. my_input.txt
+
+
+Dependencies
+------------
+        db file which must include the key-value fields:
+        "epot" with float value of DFT calculated electronic energy.
+
+    <name>.db: db file
+        file for molecules.
+        Must include the key-value-pairs:
+            "name" : str
+                identifies the catalyst.
+            "energy" or "epot" : float
+                calculated potential energy.
+
+    <name>.db: db file
+        file for surfaces and adsorbate structures.
+        Must include the key-value-pairs:
+            "name" : str
+                identifies the catalyst.
+            "energy" or "epot" : float
+                calculated potential energy.
+            "species" : str
+                adsorbate chemical formula.
+        Recommended key-value-pairs:
+            "phase" : str
+                identifies the catalyst crystal phase.
+            "facet" : str 
+                identifies the site.
+            "supercell" : str
+                formatted XxY
+                identifies the supercell.
+            "layers" : int
+                number of layers in the slab.
+
+Output
+------
+    <name>.txt : text file
+        CATMAP input file.
+
+Example
+-------
+    python generate_input.py my_input.txt
+"""
+from sys import argv
+from catmap.api.ase_data import energy_landscape
+
+# Initialize thermodynamics module.
+project = energy_landscape()
+
+# Step 1: Define search filters. These are needed to select comparable data.
+# They are typically calculation parameters, supercell sizes,
+# number of layers in slabs
+# Search strings for molecules.
+mol_select = ['vacuum=8', 'fmaxout<0.05', 'pw=500']
+
+# Step 2: Import from an ase database.
+# Import molecules.
+print('Importing molecules.')
+project.get_molecules('molecules.db', selection=mol_select, frequency_db='frequencies.db')
+
+# Search strings for all slabs, adsorbates.
+fixed_p = ['ads!=FBL', 'ads!=NEB', 'layers=5', 'pw=500', 'psp=gbrv1.5pbe',
```

Comment:
What does FBL stand for?

Reply:
Im going to delete this old tutorial script and rely on the notebook.